diff --git a/node_modules/react-native-svg/apple/Elements/RNSVGImage.mm b/node_modules/react-native-svg/apple/Elements/RNSVGImage.mm
index 5a3cb91..12237d7 100644
--- a/node_modules/react-native-svg/apple/Elements/RNSVGImage.mm
+++ b/node_modules/react-native-svg/apple/Elements/RNSVGImage.mm
@@ -45,8 +45,8 @@ @implementation RNSVGImage {
   RCTImageLoaderCancellationBlock _reloadImageCancellationBlock;
 
 #ifdef RN_FABRIC_ENABLED
-  RNSVGImageShadowNode::ConcreteState::Shared _state;
-  RCTImageResponseObserverProxy _imageResponseObserverProxy;
+  facebook::react::RNSVGImageShadowNode::ConcreteState::Shared _state;
+  facebook::react::RCTImageResponseObserverProxy _imageResponseObserverProxy;
 #endif // RN_FABRIC_ENABLED
 }
 #ifdef RN_FABRIC_ENABLED
